api/import.go

```diff
@@ -341,24 +341,45 @@ func (h ImportHandler) dependencyFromRow(fileName string, row []string) (app mod
 // Col 5: Business Service -- The name of the business service this Application should belong to.
 //                            This business service must already exist.
 //
+// Identity: May be left blank, but if provided then an identity of that kind with that name must already exist.
+// Col 6: Kind -- The kind of identity to associate with the application
+// Col 7: Name -- The name of the specific identity to associate with the application.
```

Comment:
We don't need support for the identity references.  @rromannissen has said users will be expected to _mass_ associate credentials afterwards in the UI.